### PR TITLE
add capability to specify alternalte scheduler

### PR DESCRIPTION
**Pull request by satchpx**
Signed-off-by: Sathya Balakrishnan <sathya@portworx.com>

- [x] Chart version *not* bumped (the versions are all bumped and released at the same time)
- [x] README.md updated with any new values or changes
- [ ] Updated template tests in `${CHART}/tests/*.py` 
- [ ] Updated integration tests in `${CHART}/examples/*/test/goss.yaml`

### CODE REVIEW

**Comment by satchpx** [comment]
@Crazybus Hi, could you help review this? Or perhaps assign someone to?

File: elasticsearch/README.md [hunk 1]
```diff
@@ -100,6 +100,7 @@ helm install --name elasticsearch elastic/elasticsearch --version 7.1.0 --set im
...
 | `tolerations`              | Configurable [tolerations](https://kubernetes.io/docs/concepts/configuration/taint-and-toleration/)                                                                                                                                                                                                                        | `[]`                                                                                                                      |
 | `ingress`                  | Configurable [ingress](https://kubernetes.io/docs/concepts/services-networking/ingress/) to expose the Elasticsearch service. See [`values.yaml`](./values.yaml) for an example                                                                                                                                            | `enabled: false`                                                                                                          |
+| `schedulerName`            | Name of the alternate scheduler (e.g. stork)                                                                                                | `nil`  |
```
Comment (Crazybus) [inline elasticsearch/README.md]: Could you add a link to https://kubernetes.io/docs/tasks/administer-cluster/configure-multiple-schedulers/#specify-schedulers-for-pods as something like [alternate scheduler](https://kubernetes.io/docs/tasks/administer-cluster/configure-multiple-schedulers/#specify-schedulers-for-pods). Could you also remove the `e.g. stork` just to make sure that nobody gets confused thinking that it is supported or recommended for this chart.

File: elasticsearch/values.yaml [hunk 2]
```diff
@@ -142,6 +142,11 @@ readinessProbe:
 # https://www.elastic.co/guide/en/elasticsearch/reference/current/cluster-health.html#request-params wait_for_status
 clusterHealthCheckParams: "wait_for_status=green&timeout=1s"
 
+## Use an alternate scheduler, e.g. "stork".
```
Comment (Crazybus) [inline elasticsearch/values.yaml]: Same comment as above about removing stork as an example.

File: elasticsearch/README.md (hunk 1 above)
Comment (satchpx) [reply elasticsearch/README.md]: Done

File: elasticsearch/values.yaml (hunk 2 above)
Comment (satchpx) [reply elasticsearch/values.yaml]: Done

**Comment by Crazybus** [comment]
jenkins test this please

**Comment by Crazybus** [comment]
Build is failing due to a linting error:

```
15:19:44 ==> Linting ./
15:19:44 [ERROR] templates/: render error in "elasticsearch/templates/statefulset.yaml": template: elasticsearch/templates/statefulset.yaml:48:20: executing "elasticsearch/templates/statefulset.yaml" at <.Values.schedulerNam...>: map has no entry for key "schedulerName"
```

**Comment by Crazybus** [comment]
I think setting it to an empty string is what will make the linter happiest.

```
schedulerName: ""
```

**Comment by satchpx** [comment]
Done

**Comment by Crazybus** [comment]
jenkins test this please

**Pull request merged**